### DISCUSSION TITLE
Shellcode Injection 解题讨论

### DISCUSSION THREAD

**Discussion by wumingzhilian** [HUSTSeclab/software-security-dojo#25] [General]
我们在这里讨论 Shellcode Injection 模块中遇到的问题

**Comment by RutingZhang0429** [comment]
问一下3.0，怎么只使用可见字符串编写shellcode啊

**Reply by wumingzhilian** [reply]
https://www.freebuf.com/articles/system/232280.html

**Reply by RutingZhang0429** [reply]
平台上有alpha3这些工具吗，没有的话是不是要自己手动写

**Reply by wumingzhilian** [reply]
可见字符可以本地生成，复制进去
